### DISCUSSION TITLE
AI Medics reviving players when down on advanced system

### DISCUSSION THREAD

**Discussion by SovietCharlie** [acemod/ACE3#8910] [Ideas]
Hello a newbie on Git Hub over here, not a newbie on ACE though, and, playing a lot on SP right now beacuse reasons and just noticed something about the AI that concerns me a lot.

1- AI WILL heal others team members on advanced settings even getting them up, as long as they are AI too, which, i can not really understand.

2-AI WONT heal PLAYERS on advanced settings even that those players are on the same patrol as the AI in question. Therefore my request is:

-Make the AI heal and revive players. I think it wont be so hard cause the AI already are healing others, but, in order than im not a developer but a Healthcare worker i don't know how many hours it will take, just hope you can do it at some point, it would be awesome! Thanks all of you for your time and work, keep it up!

**Comment by johnb432** [comment]
As far as I can tell, AI have been able to heal players since the medical rewrite (so before this OP was made).

> same patrol as the AI in question

What do you mean by "patrol"? For AI medics to heal injured, the injured must be in the same group as the medic.
Only AI medics can heal others (non-medic AI can heal themselves only), that's intended and by design and I doubt that's going to change.

Given it's already implemented, closing this. If you think it needs improving, ask on Discord first and we'll go from there.
